include/oneapi/dpl/pstl/execution_defs.h

```diff
@@ -186,6 +186,10 @@ template <class _ExecPolicy, class _T>
 using __enable_if_host_execution_policy = typename ::std::enable_if<
     oneapi::dpl::__internal::__is_host_execution_policy<typename ::std::decay<_ExecPolicy>::type>::value, _T>::type;
 
+template <class _ExecPolicy, class _T>
+using __enable_if_non_host_execution_policy = typename ::std::enable_if<
```

Comment:
Actually, I have several concerns regarding this new type alias `__enable_if_non_host_execution_policy`.
1. Currently, PSTL support two classes of policies - "host policies" and "hetero policies". For support each policy class there is specific implementation (on "middle" layer and backends). Theoretically, we can introduce another policy classes  - that case your code which uses `__enable_if_non_host_execution_policy` works wrong.
2) This type alias is not used in PSTL implementation just in the test, but defined in one of header of the implementation. (It is not good for repo sync activities)
3) Even a "negative" trait is useful, `!host_execution_policy` is more readable (IMHO) than `__enable_if_non_host_execution_polic`, because "!-negotiation" is the basic  C/C++ syntax.

Reply:
Fixed.